manifests/config.pp

```diff
@@ -102,7 +102,7 @@
   # Just convert to yaml
   file {"${config_path}/${config_file}":
     ensure  => $ensure,
-    content => inline_template('<%= @cobbler_config.to_yaml %>'),
```

Comment:
Could you try to use the following construction: 
content => inline_template('<%= @cobbler_config.sort.to_h.to_yaml %>'),

I want to keep it as simple as it could be